### DISCUSSION TITLE
awesome-dendron: A Dendron awesome list of extensions, vaults, and more

### DISCUSSION THREAD

**Discussion by ScriptAutomate** [dendronhq/dendron#2118] [RFCs/Ideas]
## `awesome-dendron` repository

***2022.01.31 UPDATE:*** _This discussion has resulted in the creation of the new repo: [awesome-dendron](https://github.com/dendronhq/awesome-dendron/)! Go to that repository for any further contributions_ :)

## Summary

I believe creating an [awesome-list](https://github.com/topics/awesome-list) for Dendron would be an excellent idea. Before making a repo, this discussion can be used.

Feel free to use this discussion to mention your favorites, related to your Dendron workflows, resources, extensions, and more.

- Public vaults open to contribution that you can add to your workspace
  - Official vaults
  - Community (from elsewhere)
- Recommended VS Code extensions
- Websites/blogs/etc. published using Dendron

## Awesome

> I'll actively update this section based on the comments throughout the discussion

### Dendron Official Public Vaults

> Official public vaults by Dendron. Feel free to use and contribute to them!

- Dendron Documentation | [Source](https://github.com/dendronhq/dendron-site) |  [Published Site](https://wiki.dendron.so/)
- Dendron Developer Documentation | [Source](https://github.com/dendronhq/dendron-docs) | [Published Site](https://docs.dendron.so/)
- The Open PKM Catalogue | [Source](https://github.com/dendronhq/catalogue-open-pkm) | [Published Site](https://pkm.dendron.so/)
- The Open AWS Catalogue | [Source](https://github.com/dendronhq/seeds.aws) | [Published Site](https://aws.dendron.so/)
- TLDR | [Upstream Source](https://github.com/tldr-pages/tldr) | [Source](https://github.com/kevinslin/seed-tldr) | [Published Site](https://tldr.dendron.so/)
- Dendron Templates | [Source](https://github.com/dendronhq/templates/)
- Dendron Schema Library | [Source](https://github.com/dendronhq/schema-library)

### Community Public Vaults

> Public vaults from the Dendron community. Feel free to use and contribute to them!

- [Bassman/dendron-schemas](https://github.com/Bassmann/Dendron-schemas): Collection of Dendron schema and template files

### Dendron Official VS Code / VSCodium Extensions

> Official extensions by Dendron

- [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron): The official extension that does it all!
- [Dendron Nightly/Preview](https://marketplace.visualstudio.com/items?itemName=dendron.dendron)
- [Dendron Paste Image](https://github.com/dendronhq/dendron-paste-image)
- [Dendron Markdown Shortcuts](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-shortcuts)
- [Dendron Snippet Maker](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-snippet-maker)

#### Archived / Not Recommended

> These official extensions are no longer recommended, or may cause unintended and unexpected behavior, when using Dendron

- [Dendron Markdown Preview Enhanced](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-preview-enhanced)
- [Dendron Markdown Links](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-links)

### Community VS Code / VSCodium Extensions

> Community extensions from the marketplace can potentially have conflicts (settings, shortcuts, etc.) with the official Dendron extensions, leading to unexpected behavior.

#### Markdown Enhancers

- [Markdown Writer Theme](https://marketplace.visualstudio.com/items?itemName=mgmeyers.markdown-writer-theme): VS Code theme emphasizing markdown over code
- [Markdown All-In-One](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one)
  - May cause instability or general problems with using VS Code snippets
  - Shortcuts that clash, and may need to be reassigned
    - Toggle heading (downlevel): `Ctrl + Shift + [` || `Dendron: Go Previous Sibling` (`Ctrl + Shift + [`)
    - Toggle heading (uplevel): `Ctrl + Shift + ]` || `Dendron: Go Next Sibling` (`Ctrl + Shift + ]`)
- [Todo+](https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-todo-plus)
- [Projects+ Todo+](https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-projects-plus-todo-plus)
- [Todo Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)
- [Excel to Markdown table](https://marketplace.visualstudio.com/items?itemName=csholmq.excel-to-markdown-table): Converts Excel and Google Docs spreadsheet data to Markdown table formats
- [Advanced Table Functionality](https://marketplace.visualstudio.com/items?itemName=RomanPeshkov.vscode-text-tables): Do more with markdown tables
- [vscode-reveal](https://marketplace.visualstudio.com/items?itemName=evilz.vscode-reveal): This extension lets you display a [reveal.js](https://revealjs.com/) presentation directly from an opened markdown document

#### Spellcheck, Linters, and Style Guides

- [Spell Right](https://marketplace.visualstudio.com/items?itemName=ban.spellright): Multilingual, Offline and Lightweight Spellchecker
  - Will flag `id:` values in YAML frontmatter as misspellings. Fix: `"spellright.ignoreRegExps": ["/id: .*/ig"],`
- [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
- [Write Good Linter](https://marketplace.visualstudio.com/items?itemName=travisthetechie.write-good-linter)
- [Vale](https://marketplace.visualstudio.com/items?itemName=errata-ai.vale-server)
- [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml): Validation for YAML file, such as `dendron.yml`

#### Git

- [Git Automator](https://marketplace.visualstudio.com/items?itemName=ivangabriele.vscode-git-add-and-commit): one command to commit and push all changes
- [GitDoc](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.gitdoc): Automatically commit/push/pull changes on save, so you can edit a Git repo like a multi-file, versioned document.
- [Gitlens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens): Repository/File/Line history and annotations of all your files
- [GitGraph](https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph): View a Git Graph of your repository, and easily perform Git actions from the graph. Configurable to look the way you want.
- [Path AutoComplete](https://github.com/ionutvmi/path-autocomplete): Path autocomplete for Visual Studio Code

#### Vim bindings and more

- [VSCode Vim](https://marketplace.visualstudio.com/items?itemName=vscodevim.vim): Vim key bindings 😍
- [VSCode NeoVim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim) - an alternative to VSCodeVim
- [Learn Vim](https://marketplace.visualstudio.com/items?itemName=vintharas.learn-vim): Learn Vim right within VSCode. Use this extension to learn and practice your Vim skills and become a more awesome developer.

#### Coding

- [Code Runner](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner)

#### Other

- [Auto Run Command](https://marketplace.visualstudio.com/items?itemName=gabrielgrinberg.auto-run-command): an extension that automatically runs commands after VS Code startup (e.g. `dendron.sync`)
- [Macros](https://marketplace.visualstudio.com/items?itemName=geddski.macros): Define custom macros 
- [Bookmarks](https://marketplace.visualstudio.com/items?itemName=alefragnani.Bookmarks): Bookmark lines within File
- [Vertical Limit](https://marketplace.visualstudio.com/items?itemName=generik.vertical-limit): Work with multiple cursors and blocks of text
- [CodeUI](https://marketplace.visualstudio.com/items?itemName=ryanraposo.codeui): Easier customization of every part of the VSCode UI
- [Open in Typora](https://marketplace.visualstudio.com/items?itemName=cyberbiont.vscode-open-in-typora&utm_source=VSCode.pro&utm_campaign=AhmadAwais): Open note in Typora
- [Profile Switcher](https://marketplace.visualstudio.com/items?itemName=aaronpowell.vscode-profile-switcher): Create different set of extension profiles
- [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager): Easier navigation UX between projects, directories, and workspaces
-  [Open in External App](https://marketplace.visualstudio.com/items?itemName=YuTengjing.open-in-external-app): When you want to edit your files in Typora, iAWriter or some other tool
- [Read Time](https://marketplace.visualstudio.com/items?itemName=johnpapa.read-time): Extension that provides estimated read times for your notes/docs

### Dendron Enhancers

> Scripts, tools, and repos dedicated to leveling up your Dendron powers

- [Export and Publish Draw.io Diagrams](https://github.com/LukeCarrier/dendron-publish-drawio): Use this tool to embed [draw.io / diagrams.net](https://www.diagrams.net/) diagrams in your published Dendron notes and documentation
- [`dendron-pandoc`](https://github.com/mivanit/dendron-pandoc): Run specialized [pandoc](https://pandoc.org/) filters for making Dendron links work properly, a script for adding bibliography information (or other data) to all markdown files in a vault, and more
- [`joplin2dendron`](https://github.com/chmac/joplin2dendron): Helper script to copy the correct dates from Joplin files into Dendron when migrating.

### Browser Extensions

> Extensions and add-ons for your favorite web browsers that make note taking, sharing, organizing, and collecting knowledge easier in Dendron workflows

#### Web Clippers

- [MarkDownload - Markdown Web Clipper](https://github.com/deathau/markdownload): This extension works like a web clipper, but it downloads articles in markdown format. Works on: `Chrome/Chromium` / `Firefox` / `Edge` / `Safari`
- [Web Clipper](https://chrome.google.com/webstore/detail/web-clipper/mhfbofiokmppgdliakminbgdgcmbhbac): Another markdown-format web clipper. Universal open source web clipper for Notion, OneNote, Joplin, Yuque,Bear, GitHub and more notes.
- [Convert a Website Table to Markdown](https://tabletomarkdown.com/convert-website-table-to-markdown/): A website that helps you convert HTML tables from websites into Markdown formatted pipe tables

### Dendron Showcase

> Websites published with Dendron. These users get the **Planter** [Discord role badges](https://wiki.dendron.so/notes/7c00d606-7b75-4d28-b563-d75f33f8e0d7/).

- [Kevin Lin's Garden](https://kevinslin.com/) | Discord: `@kevins8#0590`
- [Luke Carrier's Garden](https://luke.carrier.im/) | Discord: `@lukecarrier#2081`
- [Ian Jones' Garden](https://garden.ianjones.us/) | Discord: `@ianjones#3696`
- [Kevin Cunningham's Garden](https://garden.kevincunningham.co.uk/) | Discord: `@dolearning (Kevin)#3551`
- [Cameron Yik's Garden (serendipidata)](https://notes.serendipidata.com/) | Discord: `@cameron#9185`
- [Derek Ardolf's Garden (icanteven)](https://icanteven.io/) | Discord: `@icanteven#0264`
- [Adam's Garden (glucknotes)](https://glucknotes.com/) | Discord: `@glucinater21#0869`

### Other awesome lists of interest

> Awesome lists and other collections of topics related to the Dendron stack, or otherwise of interest to dendronites. These may also be candidates for pulling into a future `awesome-list` Dendron vault.

- [The Book of Secret Knowledge](https://github.com/trimstray/the-book-of-secret-knowledge)
- [Awesome git](https://github.com/dictcp/awesome-git)
- [Awesome Shell](https://github.com/alebcay/awesome-shell)
  - [Modern Unix](https://github.com/ibraheemdev/modern-unix)
- [Awesome VS Code](https://github.com/viatsko/awesome-vscode)
- [GitHub Cheat Sheet](https://github.com/tiimgreen/github-cheat-sheet)
- [Awesome GitHub Actions](https://github.com/sdras/awesome-actions)
- [Awesome TypeScript](https://github.com/dzharii/awesome-typescript)
- [Awesome Node.js](https://github.com/sindresorhus/awesome-nodejs)
- [Awesome Electron](https://github.com/sindresorhus/awesome-electron)
- [Structured Text Tools](https://github.com/dbohdan/structured-text-tools)
- [Second Brain](https://github.com/KasperZutterman/Second-Brain)
- [Digital Gardeners](https://github.com/MaggieAppleton/digital-gardeners)

### The Future

This discussion will eventually lead to an `awesome-list` in the `dendronhq` org at some point. In the meantime, it's a discussion for easy commenting, sharing, and feedback.

Most awesome lists are just one page README files of resource links, and we've thought about the idea of having a curated awesome list vault when we revisit focusing on Dendron seeds in the future. As in: it would be cool to have something like an `awesome.dendron.so` website that is a collection of awesome lists you can visit at the website, but also a public vault you can add to your workspace for easy navigation.

## An Ask

What links do you want to add? Feel free to contribute to the discussion below, and I'll update the list!

**Comment by micharris42** [comment]
Markdown All in One is definitely a must have for using VS Code as a markdown editor. It has two key bind conflicts with Dendron.

Markdown All in One: 

- Toggle heading (downlevel): `Ctrl + Shift + [`
- Toggle heading (uplevel): `Ctrl + Shift + ]`

Dendron: 

- Go Previous Sibling: `Ctrl + Shift + [`
- Go Next Sibling: `Ctrl + Shift + ]`

Users will want to edit the default key bindings when using these extensions in tandem.

**Reply by hikchoi** [reply]
It definitely adds a collection of great convenience features but a note of caution: it has been known to interfere with vscode snippets. 

I personally have even had problems with non-vscode snippet tools (like alfred) when using markdown AIO because both listen to what you type and try to insert snippets in their own way. I haven't properly spent time diagnosing these because the added features were not worth my time, but I'm sure there are ways to prevent this conflict. Would love to spend some time in the future to pinpoint what the problem was, but in the mean time I just want to let people know there might be some troubleshooting involved if you are heavily utilizing snippets in your daily workflow.

**Reply by ScriptAutomate** [reply]
I've added some updates above based on these comments :)

**Comment by rlh1994** [comment]
Another one is maybe [Project manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager) which I use to easily switch between dendron and other workspaces or folders, could also be used for easy switch between dendron workspaces if you don't want to use the command.

**Reply by ScriptAutomate** [reply]
Added!

**Comment by nickolay-kondratyev** [comment]
`Excel to Markdown table` extension: Allows copying excel/google-docs tables into Markdown format.

**Reply by ScriptAutomate** [reply]
Added!

**Comment by aleksey-rowan** [comment]
Some of  the extensions I use that haven't been mentioned yet:

- [Open in External App](https://marketplace.visualstudio.com/items?itemName=YuTengjing.open-in-external-app) - when you want to edit your files in Typora, iAWriter or some other tool
- [YAML](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) - validation for the `dendron.yml` file
- [VSCode NeoVim](https://marketplace.visualstudio.com/items?itemName=asvetliakov.vscode-neovim) - an alternative to VSCodeVim
- [Learn Vim](https://marketplace.visualstudio.com/items?itemName=vintharas.learn-vim) 😅
- [Git Graph](https://marketplace.visualstudio.com/items?itemName=mhutchie.git-graph)
- [Markdown Writer Theme](https://marketplace.visualstudio.com/items?itemName=mgmeyers.markdown-writer-theme) - theme emphasizing markdown over code
- [Read Time](https://marketplace.visualstudio.com/items?itemName=johnpapa.read-time) - who can say no to some stats?
- [Web Clipper](https://chrome.google.com/webstore/detail/web-clipper/mhfbofiokmppgdliakminbgdgcmbhbac) - another Markdown clipper

**Reply by ScriptAutomate** [reply]
Added!

**Comment by cconrad** [comment]
[Convert website table to Markdown](https://tabletomarkdown.com/convert-website-table-to-markdown) - web-based tool for the occasional conversion of tables from websites to Markdown pipe tables

**Reply by ScriptAutomate** [reply]
Added!

**Comment by ScriptAutomate** [comment]
> These are links mentioned in https://github.com/dendronhq/dendron-site/issues/359 that should be reviewed and added to the list if still applicable!

Filtering with:

```bash
grep -irn 'https://marketplace.visualstudio.com' vault/*.md | 
  sed 's:.*\https:https:g' | 
  tr -d ')>:' | sed 's:\ .*::g' | 
  sed 's:\&utm.*::g' | 
  grep -v license | 
  sort | 
  uniq 
```

I get:
```
https//marketplace.visualstudio.com/items?itemName=aaronpowell.vscode-profile-switcher
https//marketplace.visualstudio.com/items?itemName=adamwalzer.string-converter
https//marketplace.visualstudio.com/items?itemName=alefragnani.Bookmarks
https//marketplace.visualstudio.com/items?itemName=ban.spellright
https//marketplace.visualstudio.com/items?itemName=CodeStream.codestream.
https//marketplace.visualstudio.com/items?itemName=cyberbiont.vscode-open-in-typora
https//marketplace.visualstudio.com/items?itemName=d3vr.tag-search
https//marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-links
https//marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-preview-enhanced
https//marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-shortcuts
https//marketplace.visualstudio.com/items?itemName=dendron.dendron-snippet-maker
https//marketplace.visualstudio.com/items?itemName=eamodio.gitlens
https//marketplace.visualstudio.com/items?itemName=equinusocio.vsc-material-theme
https//marketplace.visualstudio.com/items?itemName=formulahendry.code-runner
https//marketplace.visualstudio.com/items?itemName=frenya.vscode-recall
https//marketplace.visualstudio.com/items?itemName=geddski.macros
https//marketplace.visualstudio.com/items?itemName=generik.vertical-limit
https//marketplace.visualstudio.com/items?itemname=github.github-vscode-theme
https//marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github
https//marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree
https//marketplace.visualstudio.com/items?itemName=ivangabriele.vscode-git-add-and-commit
https//marketplace.visualstudio.com/items?itemName=jasew.anki
https//marketplace.visualstudio.com/items?itemName=kortina.vscode-markdown-notes
https//marketplace.visualstudio.com/items?itemName=mblode.zotero
https//marketplace.visualstudio.com/items?itemName=mdickin.markdown-shortcuts
https//marketplace.visualstudio.com/items?itemName=mushan.vscode-paste-image
https//marketplace.visualstudio.com/items?itemName=notZaki.pandocciter
https//marketplace.visualstudio.com/items?itemName=orepor.color-tabs-vscode-ext
https//marketplace.visualstudio.com/items?itemName=PKief.markdown-checkbox
https//marketplace.visualstudio.com/items?itemName=ria.elastic
https//marketplace.visualstudio.com/items?itemName=RomanPeshkov.vscode-text-tables
https//marketplace.visualstudio.com/items?itemName=rxliuli.joplin-vscode-plugin
https//marketplace.visualstudio.com/items?itemName=ryanraposo.codeui
https//marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced
https//marketplace.visualstudio.com/items?itemName=Stalinbalraj.timestamp-converter
https//marketplace.visualstudio.com/items?itemName=svsool.markdown-kit
https//marketplace.visualstudio.com/items?itemName=tootone.org-mode
https//marketplace.visualstudio.com/items?itemName=vscodevim.vim
https//marketplace.visualstudio.com/items?itemName=vsls-contrib.gitdoc
https//marketplace.visualstudio.com/items?itemName=zeitnergmbh.auto-git
```

**Reply by micharris42** [reply]
There are 40 items in this list, so let's break it down a bit.

20 have already been mentioned above:

1. [Profile Switcher](https://marketplace.visualstudio.com/items?itemName=aaronpowell.vscode-profile-switcher)
2. [Bookmarks](https://marketplace.visualstudio.com/items?itemName=alefragnani.Bookmarks)
3. [Open in Typora](https://marketplace.visualstudio.com/items?itemName=cyberbiont.vscode-open-in-typora)
4. [Dendron Markdown Links](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-links)
5. [Dendron Markdown Preview Enhanced](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-preview-enhanced)
6. [Dendron Markdown Shortcuts](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-markdown-shortcuts)
7. [Dendron Snippet Maker](https://marketplace.visualstudio.com/items?itemName=dendron.dendron-snippet-maker)
8. [GitLens — Git supercharged](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
9. [Code Runner](https://marketplace.visualstudio.com/items?itemName=formulahendry.code-runner)
10. [macros](https://marketplace.visualstudio.com/items?itemName=geddski.macros)
11. [Vertical Limit](https://marketplace.visualstudio.com/items?itemName=generik.vertical-limit)
12. [Todo Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)
13. [Git Automator](https://marketplace.visualstudio.com/items?itemName=ivangabriele.vscode-git-add-and-commit)
14. [Markdown Shortcuts](https://marketplace.visualstudio.com/items?itemName=mdickin.markdown-shortcuts)
15. [Paste Image](https://marketplace.visualstudio.com/items?itemName=mushan.vscode-paste-image)
16. [Text Tables](https://marketplace.visualstudio.com/items?itemName=RomanPeshkov.vscode-text-tables)
17. [CodeUI](https://marketplace.visualstudio.com/items?itemName=ryanraposo.codeui)
18. [Markdown Preview Enhanced](https://marketplace.visualstudio.com/items?itemName=shd101wyy.markdown-preview-enhanced)
19. [Vim](https://marketplace.visualstudio.com/items?itemName=vscodevim.vim)
20. [GitDoc](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.gitdoc)

And the remaining 20 not mentioned: 

1. [Text Manipulator](https://marketplace.visualstudio.com/items?itemName=adamwalzer.string-converter)
2. [Spell Right](https://marketplace.visualstudio.com/items?itemName=ban.spellright)
3. [New Relic CodeStream: GitHub, GitLab, Bitbucket PRs and Code Review](https://marketplace.visualstudio.com/items?itemName=CodeStream.codestream)
4. [Tag Search](https://marketplace.visualstudio.com/items?itemName=d3vr.tag-search)
5. [Material Theme](https://marketplace.visualstudio.com/items?itemName=equinusocio.vsc-material-theme)
6. [Recall](https://marketplace.visualstudio.com/items?itemName=frenya.vscode-recall)
7. [GitHub Theme](https://marketplace.visualstudio.com/items?itemname=github.github-vscode-theme)
8. [GitHub Pull Requests and Issues](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github)
9. [Anki for VSCode](https://marketplace.visualstudio.com/items?itemName=jasew.anki)
10. [Markdown Notes](https://marketplace.visualstudio.com/items?itemName=kortina.vscode-markdown-notes)
11. [Citation Picker for Zotero](https://marketplace.visualstudio.com/items?itemName=mblode.zotero)
12. [Pandoc Citer](https://marketplace.visualstudio.com/items?itemName=notZaki.pandocciter)
13. [ColorTabs](https://marketplace.visualstudio.com/items?itemName=orepor.color-tabs-vscode-ext)
14. [Markdown Checkbox](https://marketplace.visualstudio.com/items?itemName=PKief.markdown-checkbox)
15. [Elasticsearch for VSCode](https://marketplace.visualstudio.com/items?itemName=ria.elastic)
16. [joplin-vscode-plugin](https://marketplace.visualstudio.com/items?itemName=rxliuli.joplin-vscode-plugin)
17. [Timestamp converter](https://marketplace.visualstudio.com/items?itemName=Stalinbalraj.timestamp-converter)
18. [Markdown Kit](https://marketplace.visualstudio.com/items?itemName=svsool.markdown-kit)
19. [Org Mode](https://marketplace.visualstudio.com/items?itemName=tootone.org-mode)
20. [auto-git](https://marketplace.visualstudio.com/items?itemName=zeitnergmbh.auto-git)

**Comment by LukeCarrier** [comment]
Some awesome lists I keep an eye on:

- https://github.com/dbohdan/structured-text-tools (mentioned in Discord sometime ago, thanks!)
- https://github.com/KasperZutterman/Second-Brain
- https://github.com/MaggieAppleton/digital-gardeners

I think it might also be worth an additional section on tools for extending Dendron that aren't part of Dendron itself? I've just published [a tool for exporting Draw.io diagrams on publish](https://github.com/LukeCarrier/dendron-publish-drawio), and I'm sure others will follow as the community grows 🌲

**Reply by ScriptAutomate** [reply]
Added!

**Comment by cconrad** [comment]
[Auto Run Command](https://marketplace.visualstudio.com/items?itemName=gabrielgrinberg.auto-run-command) - an extension that automatically runs commands after VS Code startup (e.g. `dendron.sync`)

**Reply by ScriptAutomate** [reply]
Added!

**Reply by aleksey-rowan** [reply]
Hopefully https://github.com/dendronhq/dendron/issues/2075 will happen soon and we won't need auto-run for syncing :D

**Comment by chmac** [comment]
For anyone coming from Joplin to Dendron, I've written a little helper script that I used to copy the correct dates from Joplin into Dendron. It's very **rough and ready** https://github.com/chmac/joplin2dendron.

**Reply by ScriptAutomate** [reply]
Added to `Dendron Enhancers` section :)

**Comment by aleksey-rowan** [comment]
Please add: 
- https://github.com/dendronhq/awesome-dendron/ - an awesome list for Dendron

😁

**Reply by ScriptAutomate** [reply]
Added! :grin:
